test/api/BillingTest.ts

```diff
@@ -452,14 +452,15 @@ describe('Billing Service', function() {
         await TestData.setBillingSystemValidCredentials(testData);
       });
 
-      it('should create an invoice after a transaction', async () => {
-        let response = await testData.userService.billingApi.readAll({}, TestConstants.DEFAULT_PAGING, TestConstants.DEFAULT_ORDERING, '/client/api/BillingUserInvoices');
-        const invoicesBefore = response.data.count;
+      it('should add an item to the existing invoice after a transaction', async () => {
```

Comment:
Billing Unit Test not requested in this PR
Remove all of it and create a new one and link it with the issue #1388